cime/config/e3sm/config_grids.xml

```diff
@@ -3134,6 +3167,49 @@
       <map name="GLC2LND_SMAPNAME">cpl/gridmaps/mpas.gis20km/map_mpas.gis20km_TO_fv0.9x1.25_aave.150922.nc</map>
     </gridmap>
 
+    <!-- ====================  -->
+    <!-- GLC: mpas.aisgis20km  -->
+    <!-- ====================  -->
+
+    <gridmap glc_grid="mpas.aisgis20km" lnd_grid="ne30np4">
+      <map name="LND2GLC_FMAPNAME">cpl/gridmaps/ne30np4/map_ne30np4_to_mpas.aisgis20km_aave.190403.nc</map>
+      <map name="LND2GLC_SMAPNAME">cpl/gridmaps/ne30np4/map_ne30np4_to_mpas.aisgis20km_aave.190403.nc</map>
+      <map name="GLC2LND_FMAPNAME">cpl/gridmaps/mpas.aisgis20km/map_mpas.aisgis20km_to_ne30np4_aave.190403.nc</map>
+      <map name="GLC2LND_SMAPNAME">cpl/gridmaps/mpas.aisgis20km/map_mpas.aisgis20km_to_ne30np4_aave.190403.nc</map>
+    </gridmap>
+
+    <gridmap glc_grid="mpas.aisgis20km" ocn_grid="oEC60to30v3">
+      <map name="LND2GLC_FMAPNAME">cpl/gridmaps/oEC60to30v3/map_oEC60to30v3_to_mpas.aisgis20km_aave.190403.nc</map>
+      <map name="LND2GLC_SMAPNAME">cpl/gridmaps/oEC60to30v3/map_oEC60to30v3_to_mpas.aisgis20km_bilin.190403.nc</map>
+      <map name="GLC2LND_FMAPNAME">cpl/gridmaps/mpas.aisgis20km/map_mpas.aisgis20km_to_oEC60to30v3_aave.190403.nc</map>
+      <map name="GLC2LND_SMAPNAME">cpl/gridmaps/mpas.aisgis20km/map_mpas.aisgis20km_to_oEC60to30v3_bilin.190403.nc</map>
+    </gridmap>
+
+    <gridmap glc_grid="mpas.aisgis20km" ocn_grid="oEC60to30v3wLI">
+      <map name="LND2GLC_FMAPNAME">cpl/gridmaps/oEC60to30v3/map_oEC60to30v3wLI_to_mpas.aisgis20km_aave.190713.nc</map>
+      <map name="LND2GLC_SMAPNAME">cpl/gridmaps/oEC60to30v3/map_oEC60to30v3wLI_to_mpas.aisgis20km_bilin.190713.nc</map>
+      <map name="GLC2LND_FMAPNAME">cpl/gridmaps/mpas.aisgis20km/map_mpas.aisgis20km_to_oEC60to30v3wLI_aave.190713.nc</map>
+      <map name="GLC2LND_SMAPNAME">cpl/gridmaps/mpas.aisgis20km/map_mpas.aisgis20km_to_oEC60to30v3wLI_bilin.190713.nc</map>
+    </gridmap>
+
+    <!-- ====================  -->
+    <!-- GLC: mpas.gis20km     -->
+    <!-- ====================  -->
+
+    <gridmap glc_grid="mpas.gis20km" lnd_grid="ne30np4">
+      <map name="LND2GLC_FMAPNAME">cpl/gridmaps/ne30np4/map_ne30np4_to_mpas.gis20km_mono.171002.nc</map>
```

Comment:
@jhkennedy , why are the LND2GLC and GLC2LND mapping file types (mono, intbilin) here for the `mpas.gis20km` grid different than for the `mpas.aisgis20km` above on lines 3174-3193 (aave, bilin)?

Reply:
I believe that's because of the changes discussed in this comment: 

https://github.com/E3SM-Project/E3SM/pull/3069#discussion_r303219164

@jonbob did generate the `map_ne30np4_to_gis20km_aave.181115.nc` file, but the roll back caused some conflicts I think (I could be remembering wrong). Really, I'd like @jonbob to look over all these mapping files as I had to make some of them for the `*wLI_aisgis` grid, and we might want to either regenerate them or change the method